gui/src/components/mainInput/TipTapEditor/useMainEditorWebviewListeners.ts

```diff
@@ -178,4 +181,21 @@ export function useMainEditorWebviewListeners({
     },
     [],
   );
+
+  const activeContextProviders = useAppSelector(
+    (state) => state.config.config.contextProviders,
+  );
+  const mode = useAppSelector(state=>state.session.mode);
```

Comment:
```suggestion
  const mode = useAppSelector(state => state.session.mode);
```

If you don't have auto format/linting on please do so

Reply:
turned on!
i also [added "editor.formatOnSave"](https://github.com/continuedev/continue/pull/5670/commits/c04d3c0d923d27e451cd5fce82745a9bde116a54) in our `.vscode/settings.json`

Reply:
Nice catch, thanks for doing that!